packages/camera/camera_windows/windows/capture_controller.h

```diff
@@ -180,6 +180,7 @@ class CaptureControllerImpl : public CaptureController,
   // Enumerates video_sources media types and finds out best resolution
   // for preview and video capture.
   HRESULT FindBaseMediaTypes();
+  HRESULT FindBaseMediaTypes(IMFCaptureSource* source);
```

Comment:
Please give these different names, and their own declaration comments explaining what they do. Function overloading should only be used when the methods do exactly the same thing, just with different input types.

The new method could be called `FindBaseMediaTypesForSource`.

Reply:
I have renamed the method in ff678d6e23138246d2d7553d3093b926186d2405